### PR TITLE
Update playground, and also update realtime audio visualizer and realtime chat.

### PR DESCRIPTION
**Pull request by ankit-adapt**
# Summary

In this PR we have redesigned the playground from scratch. In order to do this, we have also updated some of our realtime components.

# Changes

1. Update Realtime Visualizer. Remove dependency from `audiomotion-analyzer`.
2. Update Realtime Chat component.
3. Redesign the playground.
4. Remote react-router. (Will add some routing solution later.)

# Testing

1. Tested WebRTC example locally.
2. Tested WebSocket example locally.

# Note

In this PR if you try to send message in websocket example, it will not work. This will be fixed by #2 
Also, won't able to check WebRTC send message feature. Maybe after merging #2 we can test it.

### CODE REVIEW

**Comment by janak2** [comment]
1. You can run the webrtc example without selecting a audio or video device. One of them should be selected.
2. The audiovisualizer is not placed correctly:
Screenshot 2024-09-22 at 3 26 52 PM

**Comment by janak2** [comment]
The chat should open in a side panel similar to how it works in google meets:

Screenshot 2024-09-22 at 3 39 32 PM
Screenshot 2024-09-22 at 3 40 22 PM

**Comment by janak2** [comment]
Increase size of github and docs button

File: examples/playground-js/public/outspeed.svg
Comment (altairmn) [inline examples/playground-js/public/outspeed.svg:1]: use the logo version that works on dark background

File: examples/playground-js/src/App.tsx
Comment (altairmn) [inline examples/playground-js/src/App.tsx:1]: make the pages responsive.

**Review by altairmn — CHANGES_REQUESTED** [review]

**Comment by ankit-adapt** [comment]
@janak2 

> You can run the webrtc example without selecting a audio or video device. One of them should be selected.

Add a basic form validation logic. Now, we are checking whether audio or video device is selected. If not, then we are showing an error.

> The audiovisualizer is not placed correctly:

Make audio visualizer responsive.

>The chat should open in a side panel similar to how it works in google meets:

Updated the chat component.

>Increase size of github and docs button

Increased the size of github and docs button a little bit.

File: examples/playground-js/public/outspeed.svg
Comment (ankit-adapt) [reply examples/playground-js/public/outspeed.svg:1]: @altairmn Updated the logo.

File: examples/playground-js/src/App.tsx
Comment (ankit-adapt) [reply examples/playground-js/src/App.tsx:1]: Now pages are responsive.

File: packages/core/src/RealtimeWebSocket/RealtimeWebSocketConnection.ts [hunk 1]
```diff
@@ -214,6 +217,8 @@ export class RealtimeWebSocketConnection {
       return { error: "Failed to connect" };
     }
 
+    console.log("So", response.data);
```
Comment (altairmn) [inline packages/core/src/RealtimeWebSocket/RealtimeWebSocketConnection.ts]: What's this line?

Comment (ankit-adapt) [reply packages/core/src/RealtimeWebSocket/RealtimeWebSocketConnection.ts]: Removed! I was testing locally and forgot to remove this log.

**Review by altairmn — APPROVED** [review]

**Pull request merged**